### ISSUE TITLE
Confusion about fine-tuning

### ISSUE THREAD

**Issue by WangWenhao0716** [facebookresearch/deit#180]
I saw you set the model to eval when fine-tuning by:
`set_training_mode=args.finetune == '',  # keep in eval mode during finetuning`
I don't understand why set this in the fine-tuning stage. If the model is set to `eval` , which parameters will be changed? 
But, when I check the models' (model before tuning and after tuning) weights, they are different!!!
I'm very confused about that! Look forward to your reply. Thanks!

**Comment by HashmatShadab** [comment]
Check Issue #177

**Comment by WangWenhao0716** [comment]
But I found the models' weights (such as `patch_embed`, not dropout/BN) changed. Why? @HashmatShadab

**Comment by WangWenhao0716** [comment]
Thanks! It is very amazing!
